api/src/main/java/com/codeforcommunity/api/ITeamsProcessor.java

```diff
@@ -1,14 +1,7 @@
 package com.codeforcommunity.api;
 
 import com.codeforcommunity.auth.JWTData;
-import com.codeforcommunity.dto.team.AddGoalRequest;
-import com.codeforcommunity.dto.team.CreateTeamRequest;
-import com.codeforcommunity.dto.team.InviteUsersRequest;
-import com.codeforcommunity.dto.team.TeamDataResponse;
-import com.codeforcommunity.dto.team.TeamGoalDataResponse;
-import com.codeforcommunity.dto.team.TransferOwnershipRequest;
-import com.codeforcommunity.dto.team.UsersResponse;
-import java.util.List;
+import com.codeforcommunity.dto.team.*;
```

Comment:
We're trying to avoid using '*' inputs, can you make sure that each of the team requests is imported manually?